src/main/antora/modules/ROOT/pages/mongodb/mongo-encryption.adoc

```diff
@@ -188,7 +188,7 @@ It is possible to create custom annotations out of the provided ones.
 MongoDB Collection Info::
 +
 ====
-[source,java,indent=0,subs="verbatim,quotes",role="thrid"]
+[source,jsonc,indent=0,subs="verbatim,quotes",role="thrid"]
```

Comment:
should have been `json`, right?

Reply:
This has been intentionally because of the comments, see https://komkom.github.io/jsonc/ and https://changelog.com/news/jsonc-is-a-superset-of-json-which-supports-comments-6LwR